nixos/modules/system/boot/stage-1-init.sh

```diff
@@ -116,19 +111,40 @@ source @earlyMountScript@
 mkdir -p /tmp
 mkfifo /tmp/stage-1-init.log.fifo
 logOutFd=8 && logErrFd=9
+
+# askPassword <prompt>
+askPassword() {
+    # Ensure that we don't reset the echo flag.
+    curStatus=`stty -g`
```

Comment:
Is there a reason to use backticks instead of `$(...)`? Unless there's a reason, I think it would be better to follow what the rest of the file is already doing.

Reply:
Not really. It was an arbitrary decision, I just so happened to feel like using backticks there :p